code_search/Makefile

```diff
@@ -46,7 +46,14 @@ build-dataflow:
 			 .
 	@echo Built $(IMG)-dataflow:$(TAG)
 
-build: build-cpu build-gpu build-dataflow
+build-search-index-creator:
```

Comment:
Why do we need a new container? Why can't we use the existing Dataflow container?